newsletter2022.html

```diff
@@ -0,0 +1,601 @@
+										</p>
+									</div>							
+								</div>
+							</section>
+
+							
+							<section>
+								<a class="image">
+									<img src="images/Newsletter/fuck cancer.jpg" alt="" data-position="center center" />
+								</a>
+								<div class="content">
+									<div class="inner">
+										<header class="major">
+											<h2> Fuck Cancer Event
+											</h2>
+										</header>
+										<p>
+											We co-sponsored an event with multiple sororities and fraternities on campus. The event was hosted at Soundcheck and had a live DJ.										</p>
+									</div>							
+								</div>
+							</section>
+
+							<section>
+								<a class="image">
+									<img src="images/Newsletter/painting.jpg" alt="" data-position="center center" />
+								</a>
+								<div class="content">
+									<div class="inner">
+										<header class="major">
+											<h2> Meals on Wheels
+											</h2>
+										</header>
+										<p>
+											We partnered with Meals on Wheels to aid in distributing produce to families in need in the DC area. We also helped Meals on Wheels in its operations by helping to paint a room to be used for administrative purposes.
+										</div>							
+								</div>
+							</section>
+
+							
+							<section>
+								<a class="image">
+									<img src="images/Newsletter/ukraine.jpg" alt="" data-position="center center" />
+								</a>
+								<div class="content">
+									<div class="inner">
+										<header class="major">
+											<h2> Ukraine Hoodie Fundraiser
+											</h2>
+										</header>
+										<p>
+											We are currently selling hoodies to raise money for Ukrainian Children that were affected by the war. 
+											All profits will go straight to Tabeltochki, hoodies can be purchased at <a href="https://sigphi.typeform.com/ukraine">https://sigphi.typeform.com/ukraine</a>.
+										</p>
+									</div>
+								</div>
+							</section>
+
+
+							<section>
+								<a class="image">
+									<img src="images/Newsletter/kickball2.jpg" alt="" data-position="center center" />
+								</a>
+								<div class="content">
+									<div class="inner">
+										<header class="major">
+											<h2> Gamma Phi Moonball
+											</h2>
+										</header>
+										<p>
+											Our team won Gamma Phi's moonball tournament in support of Girls on the Run, Gamma Phi's national philanthropy event. 
+										</p>
+									</div>
+								</div>
+							</section>
+
+							<section>
+								<div class="content">
+									<div class="inner">
+										<header class="major">
+											<h2> Academic and Professional Development
+											</h2>
+										</header>
+									</div>
+								</div>
+							</section>
+
+
+
+							<section>
+								<a class="image">
+									<img src="images/Newsletter/jobs.png" alt="" data-position="center center" />

```

Comment:
maybe put a picture from a quest kid for their presentation or something and mention it in the seciton.